Source/WTF/wtf/PlatformJSCOnly.cmake

```diff
@@ -122,7 +134,12 @@ else ()
     )
 endif ()
 
-if (LOWERCASE_EVENT_LOOP_TYPE STREQUAL "glib")
+if (WIN32)
```

Comment:
Similar to the other case, does it make more sense for this to be checked in the else of having the specific event loop type set?